cfme/tests/automate/test_domain.py

```diff
@@ -276,3 +277,40 @@ def test_domain_import_git(request, appliance, url, param_type, param_value, ver
     request.addfinalizer(domain.delete_if_exists)
     assert domain.exists
     domain.delete()
+
+
+@pytest.mark.tier(1)
+def test_object_attribute_type_in_automate_schedule(appliance):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        initialEstimate: 1/15h
+        startsin: 5.8
+        tags: automate
+        testSteps:
+            1. Go to Configuration > settings > schedules
+            2. Select 'Add a new schedule' from configuration drop down
+            3. selecting 'Automation Tasks' under Action.
+            4. Select a value from the drop down list of Object Attribute Type.
+            5. Undo the selection by selecting "<Choose>" from the drop down.
+        expectedResults:
+            1.
+            2.
+            3.
+            4. No pop-up window with Internal Server Error.
```

Comment:
Line 4 and 5 are same

Reply:
Remove empty lines too

Reply:
- Line 4 and 5 are same because ```testSteps:``` 4 and 5 are expecting same result.
- I can not remove the empty lines because this is the way of writing ```expectedResults:```